webnative-examples/ditto
# Avivash/preset search

# Description

- Search presets by `name` 
- The input will stay highlighted as long as there is text in it
- If no results are found, you will see a `No search results` message
- If you've typed in the input, you can either delete the text or click the `X` to clear the search

- I also alphabetized the categories as part of this PR

**Note: I'm now seeing the duplicate Default preset. It looks like one is actually coming from the file system(possibly from that call in init), so it should be easy to remove once I add the delete functionality**

## Type of change


## Test plan (required)

- On the presets page, enter a search term, expect it to show any presets whose name matches the search term
- Type something in the input, click away, expect the border to remain highlighted
- Type something in the input that doesn't match any preset names, expect to see `No search results` message
- Type something in the input, click the X, expect the search to be cleared

## Screenshots/Screencaps

https://www.loom.com/share/adf10f98a848477f969dd86b2b009f4d
